DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -64,14 +69,20 @@ public IActionResult Index(LoginViewModel model)
                 return View("AccountNotApproved");
             }
 
-            LogIn(verifiedAdminUser, approvedDelegateUser, model.Password, model.RememberMe);
+            if (verifiedAdminUser == null && verifiedDelegateUsers.Count > 0)
+            {
+                verifiedAdminUser ??= loginService.GetVerifiedAdminUserAssociatedWithDelegateUser
+                (
+                    verifiedDelegateUsers.First(), model.Password
+                );
+            }

```

Comment:
Could we simplify this with something like:
```
var admin = verifiedAdminUser ?? loginService.GetVerifiedAdminUserAssociatedWithDelegateUser(..)
LogIn(admin, ...)
```
Note that we don't care about verifiedDelegateUsers count being > 0, because if it is 0 and admin is null, the user will be redirected to account not approved page by default.